### PR TITLE
feat: Tweak coding standards page IO-352

### PR DESCRIPTION
**Pull request by nicklem**
These are some quick edits to decouple generic improvements from the multiple coding standards task at https://github.com/codacy/docs/pull/1577

### :eyes: Live preview
https://feat-tweak-cs-page-io-352--docs-codacy.netlify.app/organizations/using-a-coding-standard/

### :construction: To do
- [x] Peer review

### CODE REVIEW

File: docs/organizations/using-a-coding-standard.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ The organization coding standard helps you ensure that Codacy analyzes multiple
 When you customize the tools or code patterns of a repository that follows the coding standard, Codacy warns you that the repository will stop following the coding standard and asks for your confirmation.
 
 !!! important
-    The coding standard enables or disables any tools that are using configuration files on the repositories following the coding standard, but these tools will keep using configuration files instead of the code pattern settings of the coding standard.
+    [Tool configuration files](../repositories-configure/configuring-code-patterns.md#using-your-own-tool-configuration-files) override coding standards.
```
Comment (nicklem) [inline docs/organizations/using-a-coding-standard.md:15]: Is this the gist? The original gives some more details, but I found it hard to follow. Let me know if I lost something!

File: docs/organizations/using-a-coding-standard.md [hunk 2]
```diff
@@ -22,23 +22,23 @@ To create a coding standard for your organization:
...
     The coding standard will only include configurations for the tools that support at least one of the selected languages.
 
     ![Selecting the languages for the coding standard](images/coding-standard-select-languages.png)
 
 1.  Configure the tools and patterns of the coding standard and click **Next: Select and apply to repositories**.
 
-    -   Toggle the tools that Codacy will run when analyzing your code
-    -   For each enabled tool, configure the code patterns that the tool will use
+    -   Toggle the tools to run when analyzing your code
+    -   For each enabled tool, configure the code patterns to use
```
Comment (nicklem) [inline docs/organizations/using-a-coding-standard.md:38]: Again, it's a bit more concise like this—unless we want to explicitly mention Codacy

**Comment by nicklem** [comment]
@prcr @claudiacarpinteiro 

Could you take a quick look when you have a moment?
See [this comment by Paulo](https://github.com/codacy/docs/pull/1577#issuecomment-1379168406) for context.

File: docs/organizations/using-a-coding-standard.md [hunk 3]
```diff
@@ -52,7 +52,7 @@ To create a coding standard for your organization:
 
 ## Setting a coding standard as default {: id="set-default"}
 
-New repositories in your organization will automatically follow the default coding standard.
+New repositories in your organization automatically follow the default coding standard.
```
Comment (prcr) [inline docs/organizations/using-a-coding-standard.md:55]: There's an opportunity here to clarify what currently happens when the single coding standard isn't defined as "default" - which is that new repositories will use the "Codacy defaults".

This could then change depending on future product changes.

File: docs/organizations/using-a-coding-standard.md (hunk 1 above)
Comment (prcr) [reply docs/organizations/using-a-coding-standard.md:15]: Your sentence is correct but omits the detail that the coding standard will still affect the on/off status of the tools using configuration files.

I'm also concerned that it loses a bit of context that may be important for users reading this page because in your version the information is stated "from the point of view" of tool configuration files, which is a big jump away from the discussion around the concept of the coding standard.

My suggestion is that you clarify the information by splitting it into separate sentences:

- The coding standard turns tools with configuration files on and off
- However, the configuration files take precedence over the code patterns defined on the coding standard

**Review by prcr — COMMENTED** [review]
The majority of the changes are clear improvements over what we had. Well done! :muscle: 

I have just one concern regarding the explanation of how the coding standard works when there are tools using configuration files, see below.

File: docs/organizations/using-a-coding-standard.md (hunk 1 above)
Comment (nicklem) [reply docs/organizations/using-a-coding-standard.md:15]: Keeping it singular for now:
```suggestion
    The coding standard turns tools with configuration files on and off. Tool configuration files, however, take precedence over the code patterns defined on the coding standard.
```

File: docs/organizations/using-a-coding-standard.md (hunk 3 above)
Comment (nicklem) [reply docs/organizations/using-a-coding-standard.md:55]:
```suggestion
New repositories in your organization automatically follow the default coding standard or Codacy defaults if no default coding standard is set.
```

**Comment by nicklem** [comment]
@prcr @claudiacarpinteiro 

Tweaked & ready for review

File: docs/organizations/using-a-coding-standard.md [hunk 4]
```diff
@@ -12,7 +12,7 @@ The organization coding standard helps you ensure that Codacy analyzes multiple
 When you customize the tools or code patterns of a repository that follows the coding standard, Codacy warns you that the repository will stop following the coding standard and asks for your confirmation.
 
 !!! important
-    The coding standard enables or disables any tools that are using configuration files on the repositories following the coding standard, but these tools will keep using configuration files instead of the code pattern settings of the coding standard.
+    The coding standard turns tools with configuration files on and off. Tool configuration files, however, take precedence over the code patterns defined on the coding standard.
```
Comment (claudiacarpinteiro) [inline docs/organizations/using-a-coding-standard.md:15]: The original sentence is definitely very complex! I like your simplified version, I only feel that we need to reinforce that we are referring to the configuration files of the previously mentioned tools.

```suggestion
    The coding standard turns tools with configuration files on and off. Those tool configuration files, however, take precedence over the code patterns defined on the coding standard.
```

**Review by claudiacarpinteiro — COMMENTED** [review]
Awesome job! 💪 I only left a small suggestion on that complex tip. 😉

File: docs/organizations/using-a-coding-standard.md (hunk 4 above)
Comment (prcr) [reply docs/organizations/using-a-coding-standard.md:15]: I agree! :+1:

**Review by prcr — APPROVED** [review]
The changes are looking good! I would just make the small change proposed by @claudiacarpinteiro before merging.
